src/coordination/coordinator_log_store.cpp

```diff
@@ -308,10 +308,15 @@ bool CoordinatorLogStore::flush() {
 bool CoordinatorLogStore::StoreEntryToDisk(const ptr<log_entry> &clone, uint64_t key_id, bool is_newest_entry) {
   buffer_serializer bs(clone->get_buf());  // data buff, nlohmann::json
   auto data_str = bs.get_str();
+  if (data_str.size() == 1 && data_str[0] == '\0') {  // empty string but bug in buffer_serializer
+    data_str = "";
+  }
```

Comment:
Same bug as before cna you explain?

Reply:
Something is terribly broken with buffer_serializer. It creates null-terminated string with 1 character which is null termination character, so JSON can't later parse it. I will add TODO around this place to investigate it more

Reply:
You can check it out here: https://godbolt.org/z/8KfodWvTT